go.mod

```diff
@@ -3,7 +3,7 @@ module github.com/GoogleCloudPlatform/alloydb-auth-proxy
 go 1.20
 
 require (
-	cloud.google.com/go/alloydbconn v1.5.2
+	cloud.google.com/go/alloydbconn v1.5.3-0.20240118193619-b19b9641fbe1
```

Comment:
Pinned to AlloyDB Go Connector public IP PR most recent commit. 

Remove after merging Go Connector dep bump